### ISSUE TITLE
`nextafter` (via both `math` and `numpy`) missing for CUDA

### ISSUE THREAD

**Issue by s-m-e** [numba/numba#9435]
As title says. Possibly related to #9424? 

Details on my [use-case can be found here](https://numba.discourse.group/t/cuda-device-function-pointers-re-implementing-scipy-integrate-solve-ivp-for-numba-cuda/2342).

**Comment by SuyashPatil98** [comment]
@s-m-e is this issue still relevant? can I work on this issue?

**Comment by s-m-e** [comment]
@SuyashPatil98 yes! Absolutely. It just went under between other projects. If you need any help, please let me know.

**Comment by Meenakshi-1802** [comment]
Hi @s-m-e,

I’d like to work on this issue if it’s still available. 
I’m a first-time contributor to this repository and excited to get started.

Thanks!

**Comment by gmarkall** [comment]
@Meenakshi-1802 Thanks for your interest! A couple of notes:

`math.nextafter` now works in Numba-CUDA. However, `np.nextafter` does not.

CUDA target development is proceeding in Numba-CUDA: https://github.com/NVIDIA/numba-cuda - so if you'd like to work on this, that repo would be the appropriate one for the PR. More info on the CUDA target future development: https://numba.readthedocs.io/en/stable/cuda/overview.html#cuda-deprecation-status

**Comment by Meenakshi-1802** [comment]
@gmarkall  Thank you for the clarification!

I’ll take a look at the numba-cuda repository and try to understand how np.nextafter can be implemented for CUDA. Please let me know if there are any specific files or areas I should focus on.

Thanks!
